src/media/js/views/homepage.js

```diff
@@ -1,10 +1,12 @@
 define('views/homepage',
-    ['core/capabilities', 'desktop_promo', 'core/format', 'core/l10n', 'core/log', 'jquery',
-     'mkt-carousel', 'core/nunjucks', 'core/requests', 'salvattore', 'core/urls', 'core/utils',
-     'utils_local', 'core/z'],
-    function(capabilities, desktopPromo, format, l10n, log, $,
-             mktCarousel, nunjucks, requests, salvattore, urls, utils,
-             utils_local, z) {
+    ['core/capabilities', 'core/format', 'core/l10n', 'core/log',
+     'core/nunjucks', 'core/requests', 'core/settings', 'core/z',
+     'desktop_promo', 'jquery', 'mkt-carousel', 'salvattore', 'core/urls',
+     'core/utils', 'utils_local', 'views/games/games'],
```

Comment:
Can we rename `views/games/games` to something more obvious? That's really awkward.